main.py

```diff
@@ -130,13 +137,23 @@ def run_experiment(dataset, compressor, top_k_accuracy, compressors_per_class, s
                     size_results.append(size_result)
 
     # todo add save in proper tabular format
+
+    #so basically i create list comprehension of all keys in first dict
+    
+    write_csv(os.path.join(os.getcwd(), 'accuracy_results.csv'), results, [x for x in results[0].keys()])
+    write_csv(os.path.join(os.getcwd(), 'speed_results.csv'), speed_results, [x for x in speed_results[0].keys()])
+    write_csv(os.path.join(os.getcwd(), 'size_results.csv'), size_results, [x for x in size_results[0].keys()])
+
+"""
```

Comment:
why is this commented? do we want to keep it or to remove it? 
I think it can be handy even if we want CSVs

Reply:
Ok, I'll uncomment it. I wasn't sure whether you wanted to keep it or not.